README.md

```diff
@@ -1,4 +1,10 @@
-<h1> XMDB <img alt="Xatafly, the logo from Xata" src="/public/flap.gif" width="50" /></h1>
+<p align="center">
+  <a href="https://github.com/xataio/xataform">
```

Comment:
Not pointing to XataForm.
And since it's the readme, doesn't make much sense to point back to the repo itself, so at a new tab, open the prod deploy sounds like a good idea to me.

```suggestion
  <a href="https://xmdb.vercel.app" target="_blank">
```

Reply:
I copied the styles of the Readme of the Xataform... 😅